app/src/main/java/com/example/recyclerviewprogram/AddUserActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.example.recyclerviewprogram;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+
+import java.io.Serializable;
+
+public class AddUserActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle saved) {
+        super.onCreate(saved);
+        setContentView(R.layout.activity_add_people);
+    }
+
+    public void save(View view){
+
+
+        EditText editname = this.findViewById(R.id.name_input);
```

Comment:
You can rename method to "initViews" and no need to contact with this, simple "findViewById(int id)"